app/frontend/wallet/shelley/shelley-transaction-planner.ts

```diff
@@ -384,7 +384,11 @@ const validateTxPlan = (txPlanResult: TxPlanResult): TxPlanResult => {
     }
   }
 
-  if (outputs.some((output) => encode(cborizeSingleTxOutput(output)).length > MAX_TX_OUTPUT_SIZE)) {
+  if (
+    outputsWithChange.some(
```

Comment:
just wondering - don't our transactions always have one output and at most one additional change output? If so, what's the point in having an "outputs" parameter?

Reply:
Hm thats a good point, I wasnt the one to introduce this, tho you are right. We also have an array for withdrawals even tho we always have one. Do u think its something worth refactoring?

Reply:
until now it was true, but I also realized now we will likely need to introduce breaking down of big outputs into multiple ones and I guess we want to do it soon, as NFTs are failing because of that, if I understood correctly, so it probably makes sense to change this tx plan computation more significantly and perhaps make such renaming (if it would still make sense) part of that